grafana-agent.rb

```diff
@@ -15,17 +15,17 @@ def install
   end
 
   def post_install
-    (etc/"grafana-cloud-agent").mkpath
+    (etc/"grafana-agent").mkpath
   end
 
   def caveats
     <<~EOS
       The agent uses a configuration file that you must customize before running:
-        #{etc}/grafana-cloud-agent/config.yml
+        #{etc}/grafana-agent/config.yml
     EOS
   end
 
-  plist_options manual: "grafana-cloud-agent -config.file=#{HOMEBREW_PREFIX}/etc/grafana-cloud-agent.yml"
+  plist_options manual: "grafana-agent -config.file=#{HOMEBREW_PREFIX}/etc/grafana-agent/config.yml"
```

Comment:
The old path name was wrong, everything else is using `#{etc}/grafana-agent/config.yml` (`#{etc}` is not available from within `plist_options` so you have to use `#{HOMEBREW_PREFIX}/etc`.)